libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -725,9 +718,9 @@ uint32_t AP_Frsky_Telem::calc_ap_status(void)
 uint32_t AP_Frsky_Telem::calc_home(void)
 {
     uint32_t home = 0;
-    Location loc;
-    Location home_loc;
-    bool get_position;
+    Location loc{};
```

Comment:
Please don't initialise things unnecessarily; further examples through this PR.

Reply:
ok but why? What's the cost of initialization?